src/checkers/inference/solver/util/NameUtils.java

```diff
@@ -0,0 +1,12 @@
+package checkers.inference.solver.util;
+
+import org.checkerframework.javacutil.AnnotationUtils;
+
+import javax.lang.model.element.AnnotationMirror;
+
+public class NameUtils {
```

Comment:
Check the usage of this Util class.

To me it seems cumbersome to encapsulate only a one line call into a class.

I would use `AnnotationUtils.annotationSimpleName(annoMirror).toString()` directly instead of calling it through a Util class.

Reply:
Ok, Since `AnnotationUtils.annotationSimpleName()` is deprecated and will be removed in upcoming releases of `javacutil`, I keep this `NameUtils` class and move the getting simple name logic from `AnnotationUtils.annotationSimpleName()` to here.